### PR TITLE
Gives the beach an edge instead of endless blackness.

### PR DESCRIPTION
**Pull request by Tastyfish**
- I mapped out a 7 tile buffer around the edge so that you can't see the edge of the world, and bushes to visually stop you.
- I also renamed `PStimer` to `timer_master`, by @tigercat2000's request.
- Adds soda dispenser to beach bar.
- Splashes

Preview

:cl:
rscadd: The beach now has a border and is splashier.
/:cl:

### CODE REVIEW

**Comment by ScratKnapp** [comment]
The chasms are quite the eyesore, in my opinion..

**Comment by Fox-McCloud** [comment]
Mind adding a soda machine to the beach's bar?

File: code/controllers/Processes/timer.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var/global/datum/controller/process/timer/PStimer
+var/global/datum/controller/process/timer/timer_controller
```
Comment (tigercat2000) [inline code/controllers/Processes/timer.dm]: ;\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_;

https://www.google.com/search?q=timer_master

**Comment by Tastyfish** [comment]
@Fox-McCloud Sure

File: code/controllers/Processes/timer.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/controllers/Processes/timer.dm]: Ahahahhaah.

:+1:

**Comment by Fox-McCloud** [comment]
Err, soda dispenser---as in what the booze dispenser is--common complain about the away mission since you can't really tend b ar effectively because of it.

**Comment by Tastyfish** [comment]
I was about to ask if you meant that. Also changing it to timer_master because that was just a joke at tiger's expense.

**Comment by FalseIncarnate** [comment]
I like the waterfall looking bit, but the chasm itself is kinda bad looking (just black squares). Other than that, nice touch adding that buffer area with trees to make it look like a nice secluded beach.

**Comment by LittleBigKid2000** [comment]
I feel like the border should be made of trees that can't be passed through rather than a chasm

**Comment by tigercat2000** [comment]
@Tastyfish the chasms look like the smoothing is a bit broken, y'know? There's weird lines running between two chasms.

**Comment by taukausanake** [comment]
I agree with @LittleBigKid2000 that the border should be trees as well. Or maybe rock formations. Didn't we have a rock border before? Kind of like a cove?

**Comment by Tastyfish** [comment]
@tigercat2000 It's actually the texture of the interior of the chasm... but i'll replace it with trees.

**Pull request merged**